templates/channels/channel_configuration.haml

```diff
@@ -432,6 +432,51 @@
     %code
       https://{{ domain }}{% url 'handlers.hub9_handler' 'delivered' object.uuid %}
 
+  -elif object.channel_type == 'FCM'
+    %h4
+      -trans "To finish configuring your Firebase Cloud Messaging connection you'll need to send the data to URL with parameters `from`, `msg` and `fcm_token` to receive messages and to register contacts use `urn`, `fcm_token` and `name` (optional)."
+
+    .clearfix
+      .config-value
+        .name
+          -trans "Name:"
+        .value {{ object.name }}
+      .config-value
+        .name
+          -trans "Address:"
+        .value {{ object.address }}
+      .config-value
+        .name
+          -trans "Notification title:"
+        .value {{ object.config_json.FCM_TITLE }}
+      .config-value
+        .name
+          -trans "Key:"
+        .value {{ object.config_json.FCM_KEY }}
+
+    .clearfix
+    %hr
+
+    %h4
+      -trans "Contact Register"
+
+    %p
+      -trans "To register contacts, is recommended that you send the data to URL below on your App."
```

Comment:
To register contacts, POST to the following URL with the parameters `urn`, `fcm_token` and optionally `name`